router/core/operation_parser.go

```diff
@@ -86,99 +114,145 @@ func (p *OperationParser) Parse(data []byte) (*ParsedOperation, error) {
 	return p.parse(data)
 }
 
+var (
+	staticOperationName = []byte("O")
```

Comment:
Could you explain why the static name is required? I didn't fully understand.

Reply:
added a descriptions